src/backend/database/schema.sql

```diff
@@ -31,7 +31,8 @@ CREATE TABLE Reviews (
     user_id INTEGER NOT NULL,
     date_of_rating DATE NOT NULL,
     comment TEXT,
-    vote INTEGER CHECK(vote = 1 OR vote = 0 OR vote = -1)
+    vote INTEGER CHECK(vote = 1 OR vote = 0 OR vote = -1),
+    UNIQUE(apt_id, user_id)

```

Comment:
If a button is not selected, should it be in the URL with the value of 0 or just not in the URL?

Reply:
Currently the design forces the user to either upvote or downvote, theres no novote.